enhancements/sig-policy/99-policy-placement-strategy/README.md

```diff
@@ -319,6 +322,149 @@ the updates, and continue enforcement rollout only when the policy on previous c
                   operator: Exist
   ```
 
+#### Story 3
+
+As a system administrator, I want to specify a placement `DecisionStrategy` bound with policy sets, allowing rollouts to
+proceed across the set on a per-cluster basis and, optionally, in the order specified.
+
+- **Summary**
+
+  - Add an option in the policy set to specify an `orderPolicies` boolean, signaling to the
+    `governance-policy-propagator` controller to automatically inject policy dependencies down the list of policies in
+    the policy set when replicating policies (i.e. policy 1 is a dependency for policy 2, policy 2 is a dependency for
+    policy 3, and so on).
+  - Add `rolloutStrategy` and `rolloutStatus` to the `PolicySet`. For a policy set, the rollout is successful for a
+    cluster only after all policies in the set become compliant.
+  - The `rolloutStatus` would be an aggregated status across all clusters and policies. The aggregated root policy
+    status would be: `Progressing` (this would include `ToApply`), `Succeeded`, or `Failed`.
```

Comment:
What is "ToApply"

Reply:
I just see that policy is waiting to be enforced.. how about "ready", ToApply is ambiguous to me. but if you ok I am good

Reply:
"ToApply" is one of the available rollout statuses: https://github.com/open-cluster-management-io/api/blob/c2b66fa4b2df8316b1165899b5eea31d237a4c70/cluster/v1alpha1/helpers.go#L25-L26

How they're used for policy is outlined earlier in this doc:
https://github.com/open-cluster-management-io/enhancements/blob/dffc91e436130c695b6191c4ee485b8e620cd21e/enhancements/sig-policy/99-policy-placement-strategy/README.md?plain=1#L114-L121

Reply:
It is defined in the Placement library--it's not something we control directly.